src/CFamily.UnitTests/Rules/DynamicCFamilyRulesConfigTests.cs

```diff
@@ -68,8 +69,7 @@ public void NullOrEmptyRulesSettings_DefaultsUsed()
             // Act
             using (new AssertIgnoreScope())
             {
-                var testSubject =
-                    new DynamicCFamilyRulesConfig(defaultConfig, settings, new TestLogger(), NoExcludedRules);
+                var testSubject = CreateTestSubject(defaultConfig, settings);
```

Comment:
Miscellaneous: changed the existing tests to use the new `CreateTestSubject` method with default parameters.